src/deepsparse/benchmark_model/README.md

```diff
@@ -14,103 +14,123 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Benchmarking
+# Benchmarking 📜
 
-The command-line (CLI) tool `deepsparse.benchmark` is meant to simplify benchmarking and be the central code path for benchmarking the engine on ONNX or SparseZoo models.
+`deepsparse.benchmark` is a command-line (CLI) tool for benchmarking our engine with ONNX models. The tool will parse the arguments, download/compile the network into the DeepSparse Engine, generate input tensors, and execute the model depending on the chosen scenario. By default, it will choose a multi-stream or asynchronous mode to optimize for throughput.
 
-## Quickstart
+### Quickstart
```

Comment:
I believe by convention we've been following markdown standards so this would be a headline 2 or ##? Are we changing it up?

Reply:
yep it's two